buildscripts/kokoro/windows.cfg

```diff
@@ -7,7 +7,7 @@ timeout_mins: 45
 # We always build mvn artifacts.
 action {
   define_artifacts {
-    regex: "**/build/test-results/**/*.xml"
+    regex: "github/grpc-java/**/build/test-results/**/sponge_log.xml"
```

Comment:
When I was debugging the netty upgrade PR having html (human readable) files was quite helpful. The xml file has the stderr output but is not easy to read and copy/paste for bug reports.

Reply:
I'm not sure why you made three comments instead of one... The stdout and stderr output is in a CDATA, so it is quite easy to copy-paste. It'd only get mangled if the text `]]>` is present). See https://source.cloud.google.com/results/invocations/7572e440-611a-457e-b023-839457fddfef/targets;query=nettytransporttest/github%2Fgrpc-java%2Fnetty%2Fbuild%2Ftest-results%2Ftest%2FTEST-io.grpc.netty.NettyTransportTest/artifacts for an example xml with stderr.

Ideally the result UI would display the logs natively, but they expect a single log file that is separate (a `sponge_log.log` IIRC).